### PR TITLE
changes

### PR DESCRIPTION
**Pull request by sergiogonzalez**

### CODE REVIEW

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/application_authorizations.jsp [hunk 1]
```diff
@@ -22,6 +22,11 @@ String redirect = ParamUtil.getString(request, "redirect");
 portletDisplay.setShowBackIcon(true);
 portletDisplay.setURLBack(redirect);
 
+// Instead of doing a return here we should have an MVCRenderCommand that checks
+// this permissions and decides whether or not we should dispatch to the
+// application_authorization.jsp or to some error.jsp, but we shouldn't have a
+// return here.
+
 if (!oAuth2AdminPortletDisplayContext.hasViewGrantedAuthorizationsPermission()) {
```
Comment (topolik) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/application_authorizations.jsp]: Thanks @sergiogonzalez, how can we check permission inside JSP when it's accessed directly using mvcPath request parameter, i.e. skip MVCRenderCommand? Make sure there's NPE in JSP and it doesn't render?

Thanks!

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes_tab2.jspf [hunk 2]
```diff
@@ -19,13 +19,16 @@
...
 	for (Map.Entry<AuthorizationModel, AssignScopesModel.Relations> scopeAliasAuthorizationModelEntry : assignScopesModel.getGlobalAuthorizationModelsRelations().entrySet()) {
 		String scopeAlias = scopeAliasAuthorizationModelEntry.getValue().getScopeAlias();
-		%>
+	%>
 
 		<div class="panel panel-secondary" id="<portlet:namespace /><%= scopeAlias %>">
 			<div class="collapse-icon panel-header panel-header-link" id="<portlet:namespace />globalAccordionHeading<%= scopeAlias %>">
 				<div class="custom-checkbox custom-control" style="display: inline-block;">
+
+					<%--We should be using aui:input instead of this label and input tags --%>
```
Comment (topolik) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes_tab2.jspf]: We tried. Then it doesn't use the new clay design and checkboxes render ugly, are not the new clay checkboxes. Is that expected?

**Comment by topolik** [comment]
Thanks a lot @sergiogonzalez for the review!

**Comment by csierra** [comment]
Pull request submitted to https://github.com/stian-sigvartsen/liferay-portal/pull/88. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes_tab2.jspf (hunk 2 above)
Comment (sergiogonzalez) [reply modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes_tab2.jspf]: Yeah, I guess that in this case we prefer all of them to be consistent so we can easily replace them all at some point with the new markup or components.

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/application_authorizations.jsp (hunk 1 above)
Comment (topolik) [reply modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/application_authorizations.jsp]: OK, so NPE covers that :)

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes_tab2.jspf (hunk 2 above)
Comment (topolik) [reply modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/assign_scopes_tab2.jspf]: Makes sense, thanks!

**Pull request closed without merging**